### ISSUE TITLE
Version on PyPI is broken?

### ISSUE THREAD

**Issue by khaledhosny** [BlackFoundryCom/black-renderer#63]
I tried `pip install blackrenderer`, but running `blackrenderer` gives an exception:
```
Traceback (most recent call last):
  File "/venv/bin/blackrenderer", line 5, in <module>
    from blackrenderer.__main__ import main
  File "/venv/lib/python3.9/site-packages/blackrenderer/__main__.py", line 5, in <module>
    from .render import renderText
  File "/venv/lib/python3.9/site-packages/blackrenderer/render.py", line 11, in <module>
    from .font import BlackRendererFont
  File "/venv/lib/python3.9/site-packages/blackrenderer/font.py", line 9, in <module>
    from fontTools.ttLib.tables.otTables import CompositeMode, PaintFormat, VariableValue
ImportError: cannot import name 'VariableValue' from 'fontTools.ttLib.tables.otTables' (/venv/lib/python3.9/site-packages/fontTools/ttLib/tables/otTables.py)
```

**Comment by justvanrossum** [comment]
Please try version "0.5.0a3": https://github.com/BlackFoundryCom/black-renderer/releases/tag/v0.5.0a3

I meant to do a more final release as soon as variability was fixed, but that didn't happen yet. Perhaps I should cut a non-alpha release sooner.

**Comment by khaledhosny** [comment]
I see, thanks. I missed the alpha releases since GitHub’s main repository page only shows the stable releases.

**Comment by ctrlcctrlv** [comment]
It's also broken on Arch due to no `python-skia` for 3.10. See https://github.com/kyamagu/skia-python/issues/170.

I managed to fix it on my system with this alternate PKGBUILD based on @alerque's:

```bash
# Maintainer: Caleb Maclennan <caleb@alerque.com>
# Hacker: Fredrick Brennan <copypaste@kittens.ph>

_pyrel=39
pkgname=python$_pyrel-skia
_pkgname=skia-python
pkgver=87.2
pkgrel=2
pkgdesc='Python binding to Skia Graphics Library (wheel)'
arch=(x86_64)
url="https://github.com/kyamagu/$_pkgname"
license=(BSD)
depends=(fontconfig opengl-driver python39)
#makedepends=(python-pip)
options=(!strip)
_py=cp$_pyrel
_wheel="${_pkgname/-/_}-$pkgver-$_py-$_py-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
source=("https://files.pythonhosted.org/packages/$_py/${_pkgname::1}/$_pkgname/$_wheel")
sha256sums=('abaa3c78850ed310a4a6b93ecd3e7b9276cd9a170defbaa6da896a2dcd1b642f')

# If anybody wants to muck around with the Chromium tree and figure out how to
# build skia from source on Arch I'm open to patches, but even after mucking
# around with ninja and Python2 and various patched bulid toolchains I have
# come up short of a way to build this against Arch packages. Drop a comment on
# the AUR page if you have ideas.

package() {
    export PIP_CONFIG_FILE=/dev/null
    warning "Using Python3.9!"
    warning 'The Python3.9 installation you use will need numpy installed separately e.g. `pip3.9 install --user numpy`!'
    python3.9 -m ensurepip
    pip3.9 install --isolated --root="$pkgdir" --ignore-installed --no-deps $_wheel
    python3.9 -O -m compileall "$pkgdir"
}
```

**Comment by khaledhosny** [comment]
This was fixed with 0.5.0 release
